app/views/partials/forms/_check_box.html.erb

```diff
@@ -0,0 +1,5 @@
+<label class="inline-flex items-center cursor-pointer mr-24 mt-6 mb-6">
+  <%= check_box_tag :field, class: "sr-only peer" %>
```

Comment:
checkbox :field doesn't make use of the "field" variable but searches for a symbol named field. Remove the ":" beforehand.